operator/pom.xml

```diff
@@ -414,6 +415,11 @@
       <artifactId>simplestub</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>commons-cli</groupId>
```

Comment:
What is this dependency? Do we need it? There is, sadly, a complicated process with CoreArch and Legal to get approval to have new dependencies. If you really need it then we can go through the process.  

If we do add it, then this needs to be added to the top-level pom following the pattern of other dependencies and have the version as a property there.

Reply:
Thanks, that's good to know. It's used by the standalone command-line utility class `DomainResourceConverter`.

Reply:
I'll add it to the top-level pom and have the version as a property in this file.